sdk/tests/conformance2/glsl3/00_test_list.txt

```diff
@@ -26,6 +26,6 @@ shader-with-invalid-characters.html
 shader-with-mis-matching-uniform-block.html
 short-circuiting-in-loop-condition.html
 texture-offset-out-of-range.html
-tricky-loop-conditions.html
+--min-version 2.0.1 tricky-loop-conditions.html
 uniform-location-length-limits.html
-vector-dynamic-indexing.html
+--min-version 2.0.1 vector-dynamic-indexing.html
```

Comment:
This one is concerning -- guaranteed support for dynamic indexing of vectors and matrices is an important ES 3.0 upgrade. Would it be possible to split off just the failing tests for 2.0.1 (vector-dynamic-indexing-bugs.html?) and enable some of them?